src/ocean/util/container/ebtree/EBTree128.d

```diff
@@ -353,16 +353,14 @@ class EBTree128 ( bool signed = false ) : IEBTree
     ***************************************************************************/
 
     private Node* add_ ( Node* node, Key key )
-    in
-    {
-        assert (node !is null, "attempted to add null node (node pool returned null?)");
-    }
     out (node_out)
     {
         assert (node_out !is null);
     }
     body
     {
+        verify (node !is null, "attempted to add null node (node pool returned null?)");
```

Comment:
Missing import?

Reply:
Yeah, curiously it compiles, though. (From the super class?)

Reply:
Yeah, I don't think it should compile with v2.071.x though. Ah, Travis is red anyway.

Reply:
Yeah, lot of import errors with new rules: https://travis-ci.org/sociomantic-tsunami/ocean/jobs/275789554#L763

Reply:
Ahh damn. I guess I should do whatever magic is required to update my local compiler.

Reply:
Though, right now only swarm and ocean will compile with dmd v2.071.x compilers, but you can use it for testing/developing.